OSDOCS-10890: Documented the 4.13.44 z-stream RNs (#77541)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.13.44 --pullspecs
----

[id="ocp-4-13-44-bug-fixes"]
==== Bug fixes

* Previously, when an `IngressController` object was configured with client SSL/TLS, but did not have the `clientca-configmap` finalizer, the Ingress Operator would try to add the finalizer without checking whether the `IngressController` object was marked for deletion. Consequently, if an `IngressController` object was configured with client SSL/TLS and was subsequently deleted, the Operator would correctly remove the finalizer. It would then repeatedly, and erroneously, try and fail to update the `IngressController` object to add the finalizer back, resulting in error messages in the Operator's logs.
+

> With this update, the Ingress Operator no longer adds the `clientca-configmap` finalizer to an `IngressController` object that is marked for deletion. As a result, the Ingress Operator no longer tries to perform erroneous updates, and no longer logs the associated errors. (link:https://issues.redhat.com/browse/OCPBUGS-34765[*OCPBUGS-34765*]

Reviewer: ```suggestion
With this update, the Ingress Operator no longer adds the `clientca-configmap` finalizer to an `IngressController` object that is marked for deletion. As a result, the Ingress Operator no longer tries to perform erroneous updates, and no longer logs the associated errors. (link:https://issues.redhat.com/browse/OCPBUGS-34765[*OCPBUGS-34765*])
```

> * Previously, the Ingress Operator would specify `spec.template.spec.hostNetwork: true` on a router deployment without specifying the `spec.template.spec.containers[*].ports[*].hostPort`. This caused the API server to set a default value for each port's `hostPort` field, which the Ingress Operator would then detect as an external update and attempt to revert it. Now, the Ingress Operator no longer incorrectly performs these updates, resolving the issue. (link:https://issues.redhat.com/browse/OCPBUGS-34409[*OCPBUGS-34409*])

Reviewer 2: Not going to block merge for this, but the asterisks here are not rendering correctly:
<img width="213" alt="image" src="https://github.com/openshift/openshift-docs/assets/3442316/c6f7a3d1-0d36-4328-8bd7-a1d7e4d1254e">
So you'll need to get a little fancier with escaping one or more of the asterisks. I think this might work:
```suggestion
* Previously, the Ingress Operator would specify `spec.template.spec.hostNetwork: true` on a router deployment without specifying the `spec.template.spec.containers[\*].ports[*].hostPort`. This caused the API server to set a default value for each port's `hostPort` field, which the Ingress Operator would then detect as an external update and attempt to revert it. Now, the Ingress Operator no longer incorrectly performs these updates, resolving the issue. (link:https://issues.redhat.com/browse/OCPBUGS-34409[*OCPBUGS-34409*])
```

> * Previously, the Ingress Operator would specify the `spec.template.spec.hostNetwork: true` on a router deployment without specifying the `spec.template.spec.containers[*].ports[*].hostPort`. This caused the API server to set a default value for each port's `hostPort` filed, which the Ingress Operator would then detect as an external update and attempt to revert it. Now, the Ingress Operator no longer incorrectly performs these updates resolving the issue. (link:https://issues.redhat.com/browse/OCPBUGS-34409[*OCPBUGS-34409*])

Reviewer: ```suggestion
* Previously, the Ingress Operator would specify `spec.template.spec.hostNetwork: true` on a router deployment without specifying the `spec.template.spec.containers[*].ports[*].hostPort`. This caused the API server to set a default value for each port's `hostPort` field, which the Ingress Operator would then detect as an external update and attempt to revert. Now, the Ingress Operator no longer incorrectly performs these updates, resolving the issue. (link:https://issues.redhat.com/browse/OCPBUGS-34409[*OCPBUGS-34409*])
```

> * Previously, the Ingress Operator was leaving the "spec.internalTrafficPolicy", "spec.ipFamilies", and "spec.ipFamilyPolicy" fields unspecified for `NodePort` and `ClusterIP` type services. The API would then set default values for these fields, which the Ingress Operator would try to revert. With this update, the Ingress Operator specifies an initial value and fixes the error caused by API default values. (link:https://issues.redhat.com/browse/OCPBUGS-33990[*OCPBUGS-33990*])

Reviewer: ```suggestion
* Previously, the Ingress Operator was leaving the `spec.internalTrafficPolicy`, `spec.ipFamilies`, and `spec.ipFamilyPolicy` fields unspecified for `NodePort` and `ClusterIP` type services. The API would then set default values for these fields, which the Ingress Operator would try to revert. With this update, the Ingress Operator specifies an initial value and fixes the error caused by API default values. (link:https://issues.redhat.com/browse/OCPBUGS-33990[*OCPBUGS-33990*])
```
